build_all.sh

```diff
@@ -14,16 +14,16 @@ curl -s https://raw.githubusercontent.com/rapidsai/gpuci-tools/main/install.sh |
 . /opt/conda/etc/profile.d/conda.sh
 conda activate rapids
 
+# Install `boa` for `mambabuild`
+conda install -yq boa
+
 # load gpuci tools
 source ~/.bashrc
 
-#conda build -c conda-forge -c defaults recipes/nvcc
-#conda build -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge -c defaults recipes/nccl
-
-conda build -c ${CONDA_USERNAME:-rapidsai} -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge --python=$PYTHON  \
+conda mambabuild -c ${CONDA_USERNAME:-rapidsai} -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge --python=$PYTHON  \
     recipes/xgboost
 
-conda build -c ${CONDA_USERNAME:-rapidsai} -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge --python=$PYTHON  \
+conda mambabuild -c ${CONDA_USERNAME:-rapidsai} -c ${NVIDIA_CONDA_USERNAME:-nvidia} -c conda-forge --python=$PYTHON  \
```

Comment:
`mambabuild` is not working fine with the `--output` flag. We should use `conda build` instead for this

Reply:
Was wondering about that. It seemed to work locally when I tried, but happy to change

Reply:
Done below